shared/src/main/scala/vyxal/ModifierFunctions.scala

```diff
@@ -0,0 +1,28 @@
+package vyxal
+
+import vyxal.Helpers.toBool
+import vyxal.Interpreter.execute
+
+object ModifierFunctions {
```

Comment:
I think all of these could just be inlined in `Builtins.scala`. They look a bit odd as methods and the only place they're going to be used is in that Builtins file anyway.

Reply:
you mean Modifiers.scala? I thought it would just keep things organized, idk